packages/cashier/src/types/types.ts

```diff
@@ -0,0 +1,14 @@
+import { PingRequest, PingResponse, VerifyEmailRequest, VerifyEmailResponse } from '@deriv/api-types';
+
+export type TSocketCallTypes = {
```

Comment:
Unfortunately, I couldn’t find a better way to do this. We need to manually add each request and response type for any call to have a type-safe socket call.

Reply:
@farzin-fs There is a file named `socket_base.js` which includes most of the Socket Calls. I think we need to define types for them and find a solution to use those types inside our stores or hooks when we want to make a socket call.

Reply:
@hamid-deriv Cool idea 👌🏻 
As we discussed over call, This can be done after #6447 is merged, Then we can also expose a `TSocketBase` type from `core` and use it in packages, Maybe you can pick it up yourself 😈